pyramid/config/views.py

```diff
@@ -1028,17 +1031,15 @@ def _check_view_options(self, **kw):
             raise ConfigurationError('Unknown view options: %s' % (kw,))
 
     def _apply_view_derivers(self, info):
-        d = pyramid.config.derivations
-        # These derivations have fixed order
+        d = pyramid.viewderivers
+
+        # These derivers are not really derivers and so have fixed order
         outer_derivers = [('attr_wrapped_view', d.attr_wrapped_view),
```

Comment:
It still feels weird to me to have something that is specific to the view machinery and HAS to exist, is applied only during the view derivation stage because it happens to need to wrap the view.

Not that I can think of a better place, but just wanted to air my feelings on it.